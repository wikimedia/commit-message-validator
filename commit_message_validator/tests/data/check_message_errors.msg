XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
YYYYYYYYYYY
YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY
Fixes:foo
Closes:bar
Task: T11
BUG:13
Bug: T11 T12

Bug: T11
Bug: T12
Bug: t13

Bug: t14
Change-Id: I1234567890abcdef123456
Change-Id: I1234567890abcdef123457
Foo bar
(cherry picked from commit 7ad8cc24d0672111cff5beddcfc9d778a1a3478e)
