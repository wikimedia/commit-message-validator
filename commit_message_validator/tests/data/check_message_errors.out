Line 1: First line should be <=80 characters
Line 2: Second line should be empty
Line 3: Line should be <=100 characters
Line 4: Use 'Bug: ' not 'Fixes:'
Line 4: Expected one space after 'Bug:'
Line 4: The bug ID is not a phabricator task ID
Line 5: Use 'Bug: ' not 'Closes:'
Line 5: Expected one space after 'Bug:'
Line 5: The bug ID is not a phabricator task ID
Line 6: Use 'Bug: ' not 'Task:'
Line 7: Expected 'Bug:' not 'BUG:'
Line 7: Expected one space after 'Bug:'
Line 7: The bug ID must be a phabricator task ID
Line 8: The bug ID is not a phabricator task ID
Line 8: Expected blank line before Bug:
Line 9: Unexpected blank line after Bug:
Line 12: The phabricator task ID must use uppercase T
Line 13: Unexpected blank line after Bug:
Line 14: The phabricator task ID must use uppercase T
Line 15: Extra Change-Id found, next at 16
Line 17: Unexpected line after Change-Id
